makefile

```diff
@@ -0,0 +1,8 @@
+data\hrtime\init:
+	python .\hrtime_migrate\migrate_sqlite.py recreate
+
+data\hrtime\migrate:
```

Comment:
And the last one, please separete this script on three different scripts, 
1) for createing tables
2) for deleting tables
3) for importing, but please use approach what was used by Luckyman in your script https://github.com/luckyman0333/ExpenseProject/blob/8a76ab19e43a5a14880f3fdb8bfb9424d714a28a/expense_manager/pythonScripts/import_xlsx.py#L172

while most of files are passed with -f key , because it enaught self-discriptive in itself. 
```
data\hrtime\migrate:
        python .\hritime_migrate\migrate -f "registrations.json" -f "cards.json" 
```

but the next step is separate this to more atomic targets. 
```
data\hrtime\migrate-regs:
        python .\hritime_migrate\migrate -f "registrations.json"
        
data\hrtime\migrate-card:
        python .\hritime_migrate\migrate -f "cards.json"                 
        
data\hrtime\migrate: data\hrtime\migrate-regs data\hrtime\migrate-card       
```
target `data\hrtime\migrate` execute both of this task. 
this approach gave us and Luckyman flexibility aporach to modify and adjust data to problem we will can resolved, and simulate what we want.